testproject/tests/test_first_step_authentication.py

```diff
@@ -16,11 +16,19 @@ def test_get_emphemeral_token(active_user_with_email_otp):
     response = login(active_user_with_email_otp)
     assert response.status_code == 200
     assert (
-        user_token_generator.check_token(response.data.get("ephemeral_token"))
+        user_token_generator.check_token(
+            user=None, token=response.data.get("ephemeral_token")
+        )
         == active_user_with_email_otp
     )
 
 
+@pytest.mark.django_db
+def test_deactivated_user(deactivated_user_with_email_otp):
+    response = login(deactivated_user_with_email_otp)
+    assert response.status_code == 400
```

Comment:
in other tests we are using `rest_framework.status` const so we can do in this test too